src/Form/FormMapper.php

```diff
@@ -136,10 +136,21 @@ public function add($name, $type = null, array $options = [], array $fieldDescri
                 $options['label'] = $this->admin->getLabelTranslatorStrategy()->getLabel($name, 'form', 'label');
             }
 
+            if (isset($options['label_translation_parameters'])) {
+                $fieldDescription->setOption('label_translation_parameters', $options['label_translation_parameters']);
+                unset($options['label_translation_parameters']);
+            }
+
             if (isset($options['help'])) {
                 $fieldDescription->setHelp($options['help']);
                 unset($options['help']);
             }
+
+
+            if (isset($options['help_translation_parameters'])) {
+                $fieldDescription->setOption('help_translation_parameters', $options['help_translation_parameters']);
+                unset($options['help_translation_parameters']);
+            }
```

Comment:
Don't you think we can replace all this code by
```suggestion
            $fieldDescription->setOptions($options);
```

Reply:
now parameters like 'help_translation_parameters' or 'label_translation_parameters' are passed like standard Symfony option in 'FormType', I dont know exactly difference between parameters `$options` and` $fieldDescriptionOptions` in `FormMapper::add` method but your proposition IMHO just copy all options to` $fieldDescriptionOptions` so one of parameters in `FormMapper::add` method will became redundant.